x-pack/plugins/observability_solution/profiling/public/components/frame_information_window/apm_transactions.tsx

```diff
@@ -239,10 +260,14 @@ export function APMTransactions({ functionName, serviceNames }: Props) {
       },
       {
         field: 'transactionSamples',
-        name: i18n.translate('xpack.profiling.apmTransactions.columns.transactionSamples', {
-          defaultMessage: 'Transaction Samples',
-        }),
-        width: '150px',
+        name: (
+          <EstimatedLabel
+            label={i18n.translate('xpack.profiling.apmTransactions.columns.transactionSamples', {
+              defaultMessage: 'Transaction Samples',
+            })}
+          />
+        ),
+        width: '152px',
```

Comment:
Nit: the `152px` is a bit too specific I am wondering if we can have the `width` consistent between the columns (also defined in a constant)?